SharedSource/UnitTests/Utilities/ArgumentUtilityTests/CheckTypeObsolete.cs

```diff
@@ -43,14 +44,14 @@ public void Succeed_ValueType ()
     [Test]
     public void Succeed_NullableValueTypeNull ()
     {
-      Assert.That (ArgumentUtility.CheckType ("arg", (object) null, typeof (int?)), Is.EqualTo (null));
+      Assert.That (ArgumentUtility.CheckType ("arg", (object?) null, typeof (int?)), Is.EqualTo (null));

```

Comment:
here the object? is okay and better than the NFO because we do not test a null-check but a type check that doesn't fail with nulls as input just because they're nulls.